packages/stark-ui/src/modules/table/components/table.component.spec.ts

```diff
@@ -254,6 +257,27 @@ describe("TableComponent", () => {
 			expect(component.sortData).toHaveBeenCalledTimes(1);
 			expect(component.orderProperties).toEqual(["test"]);
 		});
+
+		it("should display/hide the counter element when 'showRowsCounter' changes", () => {
+			const rowsCounterSelector = ".stark-table-rows-counter";
+			hostComponent.dummyData = [{ name: "dummy-data-1" }, { name: "dummy-data-2" }, { name: "dummy-data-3" }, { name: "dummy-data-4" }];
+			hostFixture.detectChanges();
+
+			let rowsCounterElement = hostFixture.debugElement.nativeElement.querySelector(rowsCounterSelector);
+			expect(rowsCounterElement).toBeNull();
+			expect(component.showRowsCounter).toBe(false);
+
+			hostComponent.showRowsCounter = true;
+			hostFixture.detectChanges();
+			expect(component.showRowsCounter).toBe(true);
+			rowsCounterElement = hostFixture.debugElement.nativeElement.querySelector(rowsCounterSelector);
+			expect(rowsCounterElement).toBeDefined();
+			expect(rowsCounterElement).not.toBeNull();
```

Comment:
You can simplify these 2 expectations in only one: `expect(rowsCounterElement).toBeTruthy();`